### ISSUE TITLE
After installing an azcli package "az" command doesn't work

### ISSUE THREAD

**Issue by MrImpossibru** [shakeelansari63/snap-packages#12]
After installing an azcli package "az" command doesn't work. "azcli" works, but this is not a default command to run azure cli.

**Comment by shakeelansari63** [comment]
That's true. And this is because `az` command alias wasn't approved by snap team.   
   
I had requested to provide `az` command alias, but it was rejected. Unfortunately they aren't providing any 2 or 3 character aliases.    
   
You can create local alias on your system In `.bashrc` or `.zshrc` file. 
```sh
alias az='azcli'
```

**Comment by MrImpossibru** [comment]
Thanks for the info. I asked on their forum to allow this alias.
https://forum.snapcraft.io/t/az-alias-for-azure-cli/4634

**Comment by lalgarvi-te** [comment]
`sudo snap alias azcli az` will create a symlink the snap way.

**Comment by shakeelansari63** [comment]
Looks like this is not going anywhere. Snap team isn't responding to forum requests.   
  
Meanwhile, we have workaround to create the alias locally using snap command.  
  
So, I will close this issue.
